.travis.yml

```diff
@@ -104,10 +104,16 @@ script:
   # Upload the docs to gh-pages.
   - |
     if [[ $PUBLISH_DOCS && ! -z "$DOCTR_DEPLOY_ENCRYPTION_KEY_NSLS_II_NSLS_II_GITHUB_IO" ]]; then
-       doctr deploy --deploy-repo NSLS-II/NSLS-II.github.io --deploy-branch-name master caproto
+        doctr deploy --deploy-repo NSLS-II/NSLS-II.github.io --deploy-branch-name master caproto
 
-       # A fix for doctr-versions-menu:
-       pip install pyparsing --upgrade
-       DEPLOY_DIR="caproto/${TRAVIS_TAG:=master}"
-       doctr deploy --command=doctr-versions-menu --build-tags --deploy-repo caproto/caproto.github.io --deploy-branch-name master ${DEPLOY_DIR}
+        # A fix for doctr-versions-menu:
+        pip install pyparsing --upgrade
+        DEPLOY_DIR="caproto/${TRAVIS_TAG:=master}"
+
+        # Avoid updating /index.html in the caproto/caproto.github.io repo.
+        # See https://goerz.github.io/doctr_versions_menu/v0.3.0/command.html#customizing-index-html
+        # for details.
+        export DOCTR_VERSIONS_MENU_WRITE_INDEX_HTML=false
+
+        doctr deploy --command=doctr-versions-menu --build-tags --deploy-repo caproto/caproto.github.io --deploy-branch-name master ${DEPLOY_DIR}
```

Comment:
Hmm, caproto docs are already working well I believe; I think what I was hoping for here was adding versions menu support in the NSLS-II docs repo.

Reply:
Oh, got it. Will try to fix it.

Reply:
Pushed a correction via a4f9db1. So, caproto.github.io goes first without any modifications, and then NSLS-II.github.io without touching `index.html`.